src/ucp/tag/tag_recv.c

```diff
@@ -17,7 +17,7 @@
 static UCS_F_ALWAYS_INLINE ucs_status_t
 ucp_tag_search_unexp(ucp_worker_h worker, void *buffer, size_t count,
                      ucp_datatype_t datatype, ucp_tag_t tag, uint64_t tag_mask,
-                     ucp_request_t *req, ucp_tag_recv_info_t *info)
+                     ucp_request_t *req, ucp_tag_recv_info_t *info, unsigned *save_rreq)
```

Comment:
can we change ucp_tag_search_unexp to return UCS_ERR_NO_ELEM when a match was not found, and UCS_INPROGRESS when found but being used (for example rndv) - to avoid the additional flag?